dojo/settings/settings.dist.py

```diff
@@ -204,6 +204,8 @@
     DD_SONARQUBE_API_PARSER_HOTSPOTS=(bool, True),
     # when enabled standard users can't change their profile information, default True
     DD_USER_PROFILE_EDITABLE=(bool, True),
+    # Feature flag for Endpoint Meta Importer
+    DD_ENDPOINT_META_IMPORTER=(bool, True),
```

Comment:
Even if it needs a db migration, I would prefer a system settings. This would be inline with other functionality we can switch off and people don't need to change their deployment to change it.

Reply:
@valentijnscholten @damiencarol thoughts on this

Reply:
Administrators can disable 17 functionalities via the UI of the system settings. `settings.dist.py` should be used to configure the installation itself (eg. authentication), not so much the functionality.

Reply:
Until now these kind of features have been flags in settings.dist.py and can be controlled via env variables. That would have my preference for this one.

Reply:
I think this kind of feature is near to questionnaires, checklists, benchmarks, jira integration, which can all be disabled via system settings.